azurerm_function_app/vars.tf

```diff
@@ -141,6 +141,12 @@ variable export_default_key {
   default = false
 }
 
+variable "app_service_plan_id" {
```

Comment:
Can you move this up before "app_service_plan_info"?

Reply:
Yes, done